src/fromager/external_commands.py

```diff
@@ -70,14 +89,17 @@ def run(
             *cmd,
         ]
 
-    logger.debug(
-        "running: %s %s in %s",
-        " ".join(f"{k}={shlex.quote(v)}" for k, v in extra_environ.items()),
-        " ".join(shlex.quote(str(s)) for s in cmd),
-        cwd or ".",
-    )
+    cmd_str = " ".join(shlex.quote(str(s)) for s in cmd)
+    env_str = " ".join(f"{k}={shlex.quote(v)}" for k, v in mask_envvars(env))
```

Comment:
The variable values are being quoted here and in `mask_envvars()`. It probably doesn't matter enough to hold this up, but it will make the output have extra quotes in some cases.

Reply:
Good catch, thanks! I have removed the quoting from this place.